.github/workflows/environment.yml

```diff
@@ -15,8 +15,30 @@ jobs:
   publish-new-environment:
     runs-on: ubuntu-20.04
     steps:
-      - name: Checkout
+      - name: (PR comment) Get PR branch
+        if: ${{ github.event_name == 'issue_comment' }}
```

Comment:
Is this right? It seems like there is no `issue_comment` trigger anymore. Same applies to `install-sh.yml`.

Reply:
Ah yeah that isn't obvious but when the trigger is `workflow_call` , the github event context is the same one as the called workflow. So this and the workflows where we call into it from another, will still have this logic because the caller workflow context event_name is `issue_comment`.

I first tried a more "refined" (or so I thought) approach where the comment trigger workflow itself would set the pending status, and final status as well, after calling the workflow. But it became messy really quick, and the status check that shows up on the PR would be "Comment Trigger", unless we passed in the name as input param to `workflow_call` and that just seemed ridiculous.
Given that our main goal with this was to reduce the number of workflows that specifically activated `on` `issue_comment`, the current design seemed the best approach.

Reply:
Ah! Yes, I didn't realize the github event context is the same. 👍